soroban-sdk/src/ledger.rs

```diff
@@ -65,6 +65,14 @@ impl Ledger {
             .into()
     }
 
+    /// Returns the maximum ledger sequence number that data can live to.
+    #[doc(hidden)]
+    pub fn max_expiration_sequence(&self) -> u32 {
```

Comment:
This is the same as `sequence`, is that intentional?

Reply:
It is not intentional. Heats really getting to me today. Thanks for catching this. Fixed in 7261049.